frontend/src/container/WorkspaceContent.jsx

```diff
@@ -475,23 +475,27 @@ export class WorkspaceContent extends React.Component {
 
     if (folderId && !folderOpen) this.handleToggleFolderOpen(folderId)
 
-    props.history.push(`${PAGE.WORKSPACE.NEW(state.workspaceIdInUrl, contentType)}?${qs.stringify(newUrlSearch, { encode: false })}&parent_id=${folderId}`)
+    props.history.push(
+      `${PAGE.WORKSPACE.NEW(state.workspaceIdInUrl, contentType)}?${qs.stringify(newUrlSearch, { encode: false })}&parent_id=${folderId}`
+    )
   }
 
   handleToggleFolderOpen = async folderId => {
     const { props, state } = this
     const folder = props.workspaceContentList.contentList.find(content => content.id === folderId) || props.workspaceShareFolderContentList.contentList.find(c => c.id === folderId)
 
+    const folderPreviousOpenState = folder.isOpen
```

Comment:
I imagine that `isOpen` is a Boolean, following the same logic wouldn't be better `isfolderPreviousStateOpen`.